docs/xml/nginx/changes.xml

```diff
@@ -5,6 +5,73 @@
 <change_log title="nginx">
 
 
+<changes ver="1.27.2" date="2024-10-02">
+
+<change type="feature">
+<para lang="ru">
+оптимизация использования памяти и уменьшение времени запуска
+в конфигурациях с большим количеством SSL-контекстов, в которых
+указаны одинаковые SSL-сертификаты, секретные ключи или списки CRL.
+</para>
+<para lang="en">
+memory usage optimization and startup speedup
+in configurations with a large number of SSL contexts
+with same SSL certificates, secret keys, or CRL lists.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+проверка клиентских сертификатов с помощью OCSP в модуле stream.
+</para>
+<para lang="en">
+client certificate validation with OCSP in the stream module.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+поддержка OCSP stapling в модуле stream.
+</para>
+<para lang="en">
+OCSP stapling support in the stream module.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+директива proxy_pass_trailers в модуле ngx_http_proxy_module.
+</para>
+<para lang="en">
+the "proxy_pass_trailers" directive in the ngx_http_proxy_module.
+</para>
+</change>
+
+<change type="change">
+<para lang="ru">
+теперь наличие директивы ssl_client_certificate не обязательно
+для проверки клиентских SSL-сертификатов.
+</para>
+<para lang="en">
+now the "ssl_client_certificate" directive is not required
+for client SSL certificates verification.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+директива ssl_client_certificate теперь поддерживает сертификаты
+с дополнительными вспомогательными данными (auxiliary trust).
+</para>
+<para lang="en">
+the "ssl_client_certificate" directive now supports certificates
+with additional auxiliary trust data.
```

Comment:
First, I'm not 100% sure we need this. Looks like a minor by-product.
Second, if we indeed need this, it should probably be moved up to other features.
Also, "additional" and "auxiliary" are synonyms, so I'd remove "additional".
Also, it should probably be mentioned that these are OpenSSL(-specific) certificates.

Maybe something like "OpenSSL trusted certificates with auxiliary information" or just "OpenSSL-specific trusted certificates".

Here's how it's described in OpenSSL sources:
https://github.com/openssl/openssl/blob/c262cc0c0444f617387adac3ed4cad9f05f9c526/include/crypto/x509.h#L146

Reply:
- This was asked at least on https://mailman.nginx.org/pipermail/nginx/2012-January/031275.html
- To best of my knowledge, changes are not necessarily grouped by a change type, but rather in a descending order of importance see e.g. changes with nginx 1.23.4. So here changes are grouped based on that trusted certificates change is of lesser importance compared to the optional "ssl_client_certificate" directive change; though I agree that both are of near equal minority and, a such, can be grouped in any order.
- Agree, "additional auxiliary" sounds like butter oil. What about "certificate with auxiliary information"? Such phrasing is repeatedly used in the OpenSSL CHANGES file, also it is more generic compared, because it includes more than just trust data (per the above link provided by you).
- As for whether this is OpenSSL-specific: this feature first appeared in OpenSSL 0.9.5, then inherited by LibreSSL and BoringSSL.  OTOH, given that such PEM label was not standardized and has no references in RFC 7468, it is valid to call it OpenSSL-specific.

I will update the patch to address this comment.